react-backend/shared/models/query_builder.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Query builder utilities
```

Comment:
Is this needed? I feel like it is taking away from the readability of our code.

Reply:
Don't you think it's nice to be able to generate simple queries dynamically? I can imagine we might want to have several slightly different views on the same data, which would currently require us to write one function for each view, instead of just making several calls to the same function:
```js
select('cbusiness', ['name', 'yob', 'gender'], { id: 1 });
select('cbusiness', ['email'], { is_email_contact_consent_granted: true });
select('cbusiness', ['phone_number'], { is_sms_contact_consent_granted: true });
// ... etc
```
You don't think the above is readable?

Reply:
I'm not completely convinced but happy to go with it 😊